payment/src/main/java/net/optile/payment/ui/redirect/RedirectUriBuilder.java

```diff
@@ -20,12 +21,12 @@
 public final class RedirectUriBuilder {
 
     /**
-     * Construct a Uri from the provided redirect
+     * Create a Uri from the provided redirect
      *
      * @param redirect to be converted into a Uri
      * @return the created Uri
      */
-    public static Uri createUri(Redirect redirect) {
+    public static Uri fromRedirect(Redirect redirect) {
         Uri uri = Uri.parse(redirect.getUrl().toString());
```

Comment:
Would
```suggestion
        Uri uri = fromURL(redirect.getUrl());
```
be better to avoid code duplication and later inconsistencies?

Reply:
The fromRedirect() method is now using the fromURL method.